jetty-servlets/src/main/java/org/eclipse/jetty/servlets/DoSFilter.java

```diff
@@ -1297,17 +1346,66 @@ public String toString()
         {
             return "RateTracker/" + _id + "/" + _type;
         }
+
+        public class Overage implements OverLimit
+        {
+            private final Duration duration;
+            private final long count;
+
+            public Overage(Duration dur, long count)
+            {
+                this.duration = dur;
+                this.count = count;
+            }
+
+            @Override
+            public RateType getRateType()
+            {
+                return _type;
+            }
+
+            @Override
+            public String getRateId()
+            {
+                return _id;
+            }
+
+            @Override
+            public Duration getDuration()
+            {
+                return duration;
+            }
+
+            @Override
+            public long getCount()
+            {
+                return count;
+            }
+
+            @Override
+            public String toString()
+            {
+                final StringBuilder sb = new StringBuilder(OverLimit.class.getSimpleName());
+                sb.append("[");
```

Comment:
Put '@' hashcode between the name and the '['

Reply:
doh!  working on it ...

Reply:
Done.